packages/swayze/lib/src/widgets/headers/gestures/header_gesture_detector.dart

```diff
@@ -250,67 +273,231 @@ class _HeaderGestureDetectorState extends State<HeaderGestureDetector> {
     );
   }
 
+  /// Handles drag starts that should start dragging a header around.
+  void handleStartDraggingHeader(
+    DragStartDetails details,
+    Range selectionRange,
+  ) {
+    setCursorState(SystemMouseCursors.basic);
+    Actions.invoke(
+      context,
+      HeaderDragStartIntent(
+        draggingPosition: details.localPosition,
+        headers: selectionRange,
+        axis: widget.axis,
+      ),
+    );
+  }
+
+  /// Handles header dragging updates.
```

Comment:
Nit: I'm not sure if this comment is adding much 😅 It's pretty much the function name. Did you forget to add something else?

Reply:
I added the explanation of why the cursor is not a closed hand.